Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM golang:1.19-alpine as builder
+FROM golang:1.20-alpine as builder
 
 COPY . /usr/src/ib-sriov-cni
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
 
-RUN apk add --update --virtual build-dependencies build-base linux-headers git && \
-    cd /usr/src/ib-sriov-cni && \
-    make clean && \
+RUN apk add --no-cache --virtual build-dependencies build-base=~0.5 linux-headers=~6.3
```

Comment:
why do we need to specify minimum version here?

Reply:
It's a requirement for hadolint check